src/test/java/com/ncc/aif/ExamplesAndValidationTest.java

```diff
@@ -455,6 +455,57 @@ void simpleHypothesisWithCluster() {
             assertAndDump(model, "simple hypothesis with cluster", seedlingValidator, true);
         }
 
+        // Create simple hypothesis with an importance value where the BUK weapon system was owned by Russia
+        @Test
+        void simpleHypothesisWithImportanceWithCluster() {
```

Comment:
Nothing needs to change here, but this would be the ideal place for the other importance changes should go...just so Bao and Darren are aware